src/stores/RoomViewStore.js

```diff
@@ -119,31 +121,33 @@ class RoomViewStore extends Store {
 
     _viewRoom(payload) {
         if (payload.room_id) {
+            const newState = {
+                roomId: payload.room_id,
+                initialEventId: payload.event_id,
+                initialEventPixelOffset: payload.event_offset,
```

Comment:
is `payload.event_offset` ever set? I don't think so. Better to use an explicit `undefined` here.